phpunit/directives/tags/wp-show.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * wp-show tag directive test.
+ */
+
+ require_once __DIR__ . '/../../../src/directives/tags/wp-show.php';
+
+ require_once __DIR__ . '/../../../src/directives/class-wp-directive-context.php';
+ require_once __DIR__ . '/../../../src/directives/class-wp-directive-processor.php';
+ 
+ require_once __DIR__ . '/../../../src/directives/wp-html.php';
+
+/**
+ * Tests for the wp-show tag directive.
+ *
+ * @group  directives
+ * @covers process_wp_show
+ */
+class Tests_Directives_WpShow extends WP_UnitTestCase {
+	public function test_directive_leaves_content_unchanged_if_when_is_true() {
+		$markup = <<<EOF
+			<wp-show when="context.myblock.open">
+				I should be shown!
+			</wp-show>
+EOF;
+		$tags = new WP_Directive_Processor( $markup );
+		$tags->next_tag();
+
+		$context_before = new WP_Directive_Context( array( 'myblock' => array( 'open' => true ) ) );
+		$context = $context_before;
```

Comment:
Should it get cloned here? Otherwise, it’s the same object referenced.
```suggestion
		$context = clone $context_before;
```

or maybe the initial context should get assigned to the variable and used in the assertion later.

Reply:
Ah, good spot! 😅 I have a few more of those in other unit tests, will file a PR to update them.